azurerm/internal/services/eventgrid/eventgrid_system_topic_event_subscription_resource_test.go

```diff
@@ -826,3 +865,74 @@ resource "azurerm_eventgrid_system_topic_event_subscription" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomString)
 }
+
+func (EventGridSystemTopicEventSubscriptionResource) identity(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-eg-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                     = "acctestacc%[3]s"
+  resource_group_name      = azurerm_resource_group.test.name
+  location                 = azurerm_resource_group.test.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+
+  tags = {
+    environment = "staging"
+  }
+}
+
+resource "azurerm_storage_queue" "test" {
+  name                 = "mysamplequeue-%[1]d"
+  storage_account_name = azurerm_storage_account.test.name
+}
+
+resource "azurerm_storage_container" "test" {
+  name                  = "vhds"
+  storage_account_name  = azurerm_storage_account.test.name
+  container_access_type = "private"
+}
+
+resource "azurerm_storage_blob" "test" {
+  name = "herpderp1.vhd"
+
+  storage_account_name   = azurerm_storage_account.test.name
+  storage_container_name = azurerm_storage_container.test.name
+
+  type = "Page"
+  size = 5120
+}
+
+resource "azurerm_eventgrid_system_topic_event_subscription" "test" {
+  name                = "acctesteg-%[1]d"
+  system_topic        = azurerm_eventgrid_system_topic.test.name
```

Comment:
This test is failing because this resource doesn't exist
```
testing.go:620: Step 1/6 error: Error running pre-apply refresh: exit status 1
Error: Reference to undeclared resource
on terraform_plugin_test.tf line 46, in resource "azurerm_eventgrid_system_topic_event_subscription" "test":
46:   system_topic        = azurerm_eventgrid_system_topic.test.name
A managed resource "azurerm_eventgrid_system_topic" "test" has not been
declared in the root module.
--- FAIL: TestAccEventGridSystemTopicEventSubscription_identity (16.50s)
```